lexicons/com/atproto/admin/defs.json

```diff
@@ -2,6 +2,14 @@
   "lexicon": 1,
   "id": "com.atproto.admin.defs",
   "defs": {
+    "statusAttr": {
```

Comment:
Swapped out "state" for "status" (inspired by Foysal's PR https://github.com/bluesky-social/atproto/pull/1617)

As well, differentiated between the names of these two:
"status attribute": an individual attribute of a subject's status. For instance, it's current takedown status
"status": all attributes of a subject's status